brainstormforce/astra
# AST-2995 - Improvement - Added Tiktok as a Social Icon

### Description

 - In header or footer builder, we do not have TikTok as an option. We need to add tiktok.
 - https://brainstormforce.atlassian.net/browse/AST-2995

### Screenshots

[Astra WordPress Theme Community | Facebook](https://www.facebook.com/groups/wpastra/posts/1533785357090723/)

### Types of changes




- Improvements

### How has this been tested?

- Choose Social Widget on Header or Footer Builder
- Choose TikTok as icon
- View in frontend
- Check by adding URL as well

### Checklist:
